superset-frontend/src/explore/components/SaveModal.tsx

```diff
@@ -244,17 +258,15 @@ class SaveModal extends React.Component {
   }
 }
 
-SaveModal.propTypes = propTypes;
-
-function mapStateToProps({ explore, saveModal }) {
+function mapStateToProps({ explore, saveModal }: Record<string, any>) {
   return {
     datasource: explore.datasource,
     slice: explore.slice,
     can_overwrite: explore.can_overwrite,
     userId: explore.user_id,
     dashboards: saveModal.dashboards,
     alert: saveModal.saveModalAlert,
-  };
+  } as Partial<SaveModalProps>;
```

Comment:
instead of casting the response, can we define the function to return a `Partial<SaveModalProps>` instead?